src/utils.rs

```diff
@@ -42,6 +49,15 @@ impl AsRef<CurvePoint> for CurvePoint {
 }
 
 impl CurvePoint {
+    pub fn convert_to_projective_point(&self) -> FieldBytes {
+        self.0.to_affine().x()
+    }
```

Comment:
issue: This function does not return a `ProjectivePoint`. Please pick a different name.

[Affine](https://docs.rs/k256/latest/k256/struct.AffinePoint.html) and [projective](https://docs.rs/k256/latest/k256/struct.ProjectivePoint.html) are different representations for points on an elliptic curve. They have different properties, some of which are more useful for different types of computations; this is why k256 supports both representations. 

[The docs for the `x` method](https://docs.rs/elliptic-curve/0.13.4/elliptic_curve/point/trait.AffineCoordinates.html#tymethod.x) explain that it gets the x-coordinate from the affine representation. The "projection" in the variable name is saying that we're projecting the curve point down to the x-axis (to get just that coordinate), not that we're converting to a projective representation.

Reply:
I'd prefer a name that focuses on the fact that we're getting the x-coordinate. Just `x()` would be okay, or `x_coordinate()`. The paper calls it "the projection of the curve point onto the x-axis", which is where the name `x_projection() came from, but if you think it's confusing to use "projection" when it's the affine representation, maybe `x_affine()`.